app/assets/stylesheets/layout.scss

```diff
@@ -246,6 +246,11 @@ table tbody td[align="right"], table tbody th[align="right"], table thead th[ali
   word-break: break-all;
 }
 
+.live-red-text {
```

Comment:
You can integrate with changes merged into master now Aaron.  Class name is the same

Reply:
ok, deleted the second entry so we now just have the one